apps/src/sites/studio/pages/rubrics/new.js

```diff
@@ -8,12 +8,14 @@ $(document).ready(() => {
   const unitName = lessonData.unitName;
   const lessonNumber = lessonData.lessonNumber;
   const levels = lessonData.levels;
+  const lessonId = lessonData.id;
```

Comment:
nit: you can do `const {unitName, lessonNumber, levels, lessonId} = lessonData` here to make this a bit more succinct